ro_help/hub/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.8.0/css/bulma.min.css">
     <link rel="stylesheet" href="{% static "css/bulma-collapsible.min.css" %}">
     <link rel="stylesheet" href="{% static "css/hub.css" %}">
+    <link rel="shortcut icon" href="{% static "images/favicons/favicon.ico" %}"/>
```

Comment:
we should have all the favicons ( for all dimensions ) - you can find them [here](https://github.com/code4romania/covid-assets/tree/master/favicons/RO-help)

Reply:
now i saw that you have the manifest and the browserconfig in the assets repo. ok. i change it.